traits/src/career.rs

```diff
@@ -6,9 +6,22 @@ use serde::{Deserialize, Serialize};
 #[derive(Encode, Decode, Debug, Clone, Copy, Eq, PartialEq, TypeInfo, MaxEncodedLen)]
 #[cfg_attr(feature = "std", derive(Serialize, Deserialize))]
 pub enum CareerType {
+	HackerWizard,
 	HardwareDruid,
 	RoboWarrior,
 	TradeNegotiator,
-	HackerWizard,
 	Web3Monk,
 }
+
+impl CareerType {
+	pub fn from_u8(num: u8) -> Option<CareerType> {
```

Comment:
Why is it required? I see the only call is from the unit test

Reply:
I had to use this for this part of the code.

https://github.com/Phala-Network/rmrk-substrate/blob/incubation-pallet/pallets/phala-world/src/incubation.rs#L282-L287

Reply:
Removed